frontend/src/components/pages/cabins/Admin/InlineTableCell.tsx

```diff
@@ -0,0 +1,15 @@
+import { TableCell } from "@material-ui/core";
+
+type TableCellProps = {
+  children: React.ReactNode;
+};
+
+const InlineTableCell: React.VFC<TableCellProps> = ({ children }) => {
```

Comment:
`React.VFC` is just a functional component that does not accept `children`

```suggestion
const InlineTableCell: React.FC = ({ children }) => {
```